db/view/view_update_generator.cc

```diff
@@ -166,7 +228,11 @@ void view_update_generator::setup_metrics() {
 
         sm::make_gauge("sstables_to_move_count",
                 sm::description("Number of sets of sstables which are already processed and wait to be moved from their staging directory"),
-                [this] { return _sstables_to_move.size(); })
+                [this] { return _sstables_to_move.size(); }),
+
+        sm::make_gauge("sstables_pending_work",
+                sm::description("Number of bytes to be processed from staging SSTables. The bytes of SSTables already processed is subtracted from the result."),
```

Comment:
Isn't it simpler to just say "Number of bytes remaining to be processed from staging SSTables"?
To be honest, I doubt that any user will know what "staging SSTables" are. This goes back to my usual thinking that our descriptions of metrics is not useful and we need much longer descriptions, in the form of a document - not a single sentence.

Reply:
I think this is inaccurate: this describes the number of bytes remaining to be processed in the current iteration of the view update generator loop. Since that iteration started more staging sstables could have been added and this will not be reflected in this metric. Also, there is a [PR](https://github.com/scylladb/scylladb/pull/11745) which will not consume all staging sstables on each iteration.

Reply:
> Isn't it simpler to just say "Number of bytes remaining to be processed from staging SSTables"? To be honest, I doubt that any user will know what "staging SSTables" are. This goes back to my usual thinking that our descriptions of metrics is not useful and we need much longer descriptions, in the form of a document - not a single sentence.

Will drop the Staging adjective. Probably better to just use SSTables, and say the reason we're processing those bytes.

 "Number of bytes remaining to be processed from SSTables for view updates"

Reply:
> I think this is inaccurate: this describes the number of bytes remaining to be processed in the current iteration of the view update generator loop. Since that iteration started more staging sstables could have been added and this will not be reflected in this metric. Also, there is a [PR](https://github.com/scylladb/scylladb/pull/11745) which will not consume all staging sstables on each iteration.

But I am actually registering all sstables to the progress tracker, not only the ones being currently processed. See callers of on_sstable_registration().

Reply:
I see. That indeed makes this better.